bundles/org.openhab.binding.linkplay/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,473 @@
+			<channel id="sample-rate" typeId="sampleRate"/>
+			<channel id="bit-depth" typeId="bitDepth"/>
+			<channel id="bit-rate" typeId="bitRate"/>
+			<channel id="current-playlist-name" typeId="playlistName">
+				<label>Current Playlist Name</label>
+				<description>Name of the current playlist</description>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="input">
+		<label>Source Selection</label>
+		<description>Input and connection related channels</description>
+		<channels>
+			<channel id="source" typeId="sourceInput"/>
+			<channel id="bluetoothConnected" typeId="bluetoothConnected"/>
+			<channel id="bluetooth-paired-device" typeId="bluetoothPairedDevice"/>
+			<channel id="line-in-active" typeId="lineInActive"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="equalizer">
+		<label>Equalizer &amp; Output</label>
+		<description>Equalizer settings and audio output</description>
+		<channels>
+			<channel id="enabled" typeId="eqEnabled"/>
+			<channel id="preset" typeId="eqPreset"/>
+			<channel id="band" typeId="eqBand"/>
+			<channel id="output-hardware-mode" typeId="outputHardwareMode"/>
+			<channel id="channel-balance" typeId="channelBalance"/>
+			<channel id="spdif-switch-delay-ms" typeId="spdifSwitchDelayMs"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="multiroom">
+		<label>Multi-room</label>
+		<description>Group playback features</description>
+		<channels>
+			<channel id="volume" typeId="slaveVolume"/>
+			<channel id="mute" typeId="slaveMute"/>
+			<channel id="active" typeId="multiroomActive"/>
+			<channel id="join" typeId="joinGroup"/>
+			<channel id="leave" typeId="leaveGroup"/>
+			<channel id="manage" typeId="addRemoveMember"/>
+			<channel id="ungroup" typeId="ungroup"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="device">
+		<label>Device</label>
+		<description>Physical device settings</description>
+		<channels>
+			<channel id="led-enabled" typeId="ledEnabled"/>
+			<channel id="touch-keys-enabled" typeId="touchKeysEnabled"/>
+			<channel id="shutdown-timer" typeId="shutdownTimer"/>
+			<channel id="reboot" typeId="reboot"/>
+			<channel id="factory-reset" typeId="factoryReset"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="presets">
+		<label>Presets</label>
+		<description>Stored presets</description>
+		<channels>
+			<channel id="count" typeId="presetCount"/>
+			<channel id="play" typeId="playPreset"/>
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="preset">
+		<label>Preset</label>
+		<description>Information stored for a single device preset</description>
+		<channels>
+			<channel id="name" typeId="presetName"/>
+			<channel id="url" typeId="presetUrl"/>
+			<channel id="source" typeId="presetSource"/>
+			<channel id="pic-url" typeId="presetPicUrl"/>
+			<channel id="pic" typeId="presetPic"/>
+			<channel id="play" typeId="presetPlay"/>
```

Comment:
Applies to all channel id's ( i think they are good) and channel type id's. Please stick to the naming co9nvention `lower-case-hyphen`